docs/api/04_customers/create-update-customer.mdx

```diff
@@ -273,6 +279,7 @@ If the customer already exists, the call will work as an update
 | -----------| -----| ----------- |
 | payment_provider | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Payment provider used to bill the customer.<br/>Only accepted value: `stripe`, this field is required if you want to assign a `provider_customer_id`
 | provider_customer_id | String &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Customer ID on the payment provider.<br/>If not provided, Lago can optionally create the provider customer
+| sync_with_provider | Boolean &nbsp &nbsp &nbsp<Optional>**Optional**</Optional> | Field that determines whether to create customer in payment provider or not
```

Comment:
I don't understand the difference between `sync_with_provider` and `sync`. Please can you explain?

Reply:
`sync_with_provider` is a property that determines if customer will also be created in gocardless/stripe after creation in Lago. On the other hand, `sync` is only tech related field. It specifies if creation in payment provider will be handled in the sync/async way, but this field has no major effect on the 'product'